<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>763</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>50</horstretch>
    <verstretch>50</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>20000</width>
    <height>20000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>交通信号控制系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="interface.qrc">
    <normaloff>:/Files/images/logo.jpg</normaloff>:/Files/images/logo.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>850</width>
     <height>705</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>500</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::West</enum>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Triangular</enum>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>  系统及管理  </string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>-40</x>
              <y>0</y>
              <width>141</width>
              <height>561</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="btnUserIn">
               <property name="minimumSize">
                <size>
                 <width>105</width>
                 <height>55</height>
                </size>
               </property>
               <property name="text">
                <string>用户登录</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnUserLog">
               <property name="minimumSize">
                <size>
                 <width>105</width>
                 <height>55</height>
                </size>
               </property>
               <property name="text">
                <string>用户日志</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnUserManage">
               <property name="minimumSize">
                <size>
                 <width>105</width>
                 <height>55</height>
                </size>
               </property>
               <property name="text">
                <string>用户管理</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSysPara">
               <property name="minimumSize">
                <size>
                 <width>105</width>
                 <height>55</height>
                </size>
               </property>
               <property name="text">
                <string>系统参数</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>节点参数及状态</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>-10</x>
              <y>0</y>
              <width>107</width>
              <height>541</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPushButton" name="btnNodePara">
               <property name="minimumSize">
                <size>
                 <width>105</width>
                 <height>55</height>
                </size>
               </property>
               <property name="text">
                <string>节点信息</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnPhase">
               <property name="minimumSize">
                <size>
                 <width>105</width>
                 <height>55</height>
                </size>
               </property>
               <property name="text">
                <string>相位灯组</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnConfict">
               <property name="minimumSize">
                <size>
                 <width>105</width>
                 <height>55</height>
                </size>
               </property>
               <property name="text">
                <string>冲突相位</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnStage">
               <property name="minimumSize">
                <size>
                 <width>105</width>
                 <height>55</height>
                </size>
               </property>
               <property name="text">
                <string>阶段配时</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnDetector">
               <property name="minimumSize">
                <size>
                 <width>105</width>
                 <height>55</height>
                </size>
               </property>
               <property name="text">
                <string>检测器</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnStatus">
               <property name="minimumSize">
                <size>
                 <width>105</width>
                 <height>55</height>
                </size>
               </property>
               <property name="text">
                <string> 状态及控制</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnTrafficData">
               <property name="minimumSize">
                <size>
                 <width>105</width>
                 <height>55</height>
                </size>
               </property>
               <property name="text">
                <string>交通数据统计</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnEventLog">
               <property name="minimumSize">
                <size>
                 <width>105</width>
                 <height>55</height>
                </size>
               </property>
               <property name="text">
                <string>事件日志</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>线面协调</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>-40</x>
              <y>0</y>
              <width>141</width>
              <height>251</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QPushButton" name="btnTimePlan">
               <property name="minimumSize">
                <size>
                 <width>105</width>
                 <height>55</height>
                </size>
               </property>
               <property name="text">
                <string>  群调度计划</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnCluster">
               <property name="minimumSize">
                <size>
                 <width>105</width>
                 <height>55</height>
                </size>
               </property>
               <property name="text">
                <string>    群设置</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>VIP线路及外设</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>-10</x>
              <y>10</y>
              <width>107</width>
              <height>281</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QPushButton" name="btnVip">
               <property name="minimumSize">
                <size>
                 <width>105</width>
                 <height>55</height>
                </size>
               </property>
               <property name="text">
                <string>VIP设置查看</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnTimer">
               <property name="minimumSize">
                <size>
                 <width>105</width>
                 <height>55</height>
                </size>
               </property>
               <property name="text">
                <string>倒计时</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnFlag">
               <property name="minimumSize">
                <size>
                 <width>105</width>
                 <height>55</height>
                </size>
               </property>
               <property name="text">
                <string>可变标志</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="logo">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>140</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100000</width>
            <height>1000000</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <widget class="QFrame" name="logoframe">
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>10</y>
             <width>100</width>
             <height>85</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>100</y>
             <width>101</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>人相吉信号控制系统
          网络版</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="leftWindow" native="true">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>410</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>685</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>6</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item>
         <widget class="Line" name="diveLine">
          <property name="minimumSize">
           <size>
            <width>700</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="rightWindow" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>850</width>
            <height>500</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="debugWindow">
          <property name="minimumSize">
           <size>
            <width>700</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>222220</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>系统</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>显示及设置</string>
    </property>
    <addaction name="action_NodeInfo"/>
    <addaction name="action_Phase"/>
    <addaction name="action_Conflict"/>
    <addaction name="action_Schedule"/>
    <addaction name="action_Plan"/>
    <addaction name="action_Stage"/>
    <addaction name="action_Dec"/>
    <addaction name="action_Status"/>
    <addaction name="action_Note"/>
    <addaction name="action_12"/>
    <addaction name="action_13"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>窗口</string>
    </property>
    <addaction name="dispDebugWindow"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_16"/>
    <addaction name="action_17"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>15</height>
    </size>
   </property>
  </widget>
  <action name="action">
   <property name="text">
    <string>用户登录</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>用户管理</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="action_NodeInfo">
   <property name="text">
    <string>节点信息</string>
   </property>
  </action>
  <action name="action_Phase">
   <property name="text">
    <string>相位灯组</string>
   </property>
  </action>
  <action name="action_Conflict">
   <property name="text">
    <string>相位屏蔽</string>
   </property>
  </action>
  <action name="action_Plan">
   <property name="text">
    <string>时基表</string>
   </property>
  </action>
  <action name="action_Stage">
   <property name="text">
    <string>阶段配时</string>
   </property>
  </action>
  <action name="action_Dec">
   <property name="text">
    <string>检测器</string>
   </property>
  </action>
  <action name="action_Status">
   <property name="text">
    <string>状态及控制</string>
   </property>
  </action>
  <action name="action_Note">
   <property name="text">
    <string>事件日志</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>日志及用户管理</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>倒计时及可变标志</string>
   </property>
  </action>
  <action name="dispDebugWindow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示调试窗口</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>参考文档</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>关于...</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>基本属性设置</string>
   </property>
  </action>
  <action name="action_19">
   <property name="text">
    <string>时间计划设置</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>阶段设置</string>
   </property>
  </action>
  <action name="action_21">
   <property name="text">
    <string>相位设置</string>
   </property>
  </action>
  <action name="action_22">
   <property name="text">
    <string>通道参数设置</string>
   </property>
  </action>
  <action name="action_23">
   <property name="text">
    <string>检测器参数设置</string>
   </property>
  </action>
  <action name="action_24">
   <property name="text">
    <string>设备模块参数设置</string>
   </property>
  </action>
  <action name="action_25">
   <property name="text">
    <string>倒计时及可变标志设置</string>
   </property>
  </action>
  <action name="action_26">
   <property name="text">
    <string>节点系统管理</string>
   </property>
  </action>
  <action name="action_27">
   <property name="text">
    <string>群属性设置</string>
   </property>
  </action>
  <action name="action_28">
   <property name="text">
    <string>域属性设置</string>
   </property>
  </action>
  <action name="actionVIP_2">
   <property name="text">
    <string>VIP属性设置</string>
   </property>
  </action>
  <action name="action_29">
   <property name="text">
    <string>系统参数设置</string>
   </property>
  </action>
  <action name="action_30">
   <property name="text">
    <string>优化算法设置</string>
   </property>
  </action>
  <action name="action_Schedule">
   <property name="text">
    <string>时段表</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="interface.qrc"/>
 </resources>
 <connections/>
</ui>
